src/containers/NFT/NFTHeader/styles.scss

```diff
@@ -0,0 +1,401 @@
+@import '../../shared/css/variables.scss';
+
+.section {
+  margin: 0 32px;
+  @include for-size(tablet-landscape-up) {
+    width: 552px;
+    margin: 0px auto;
+  }
+
+  @include for-size(desktop-up) {
+    width: 1140px;
+  }
+}
+
+table {
+  margin-top: 24px;
+  font-size: 14px;
+
+  .col1 {
+    width: 180px;
```

Comment:
having explicit sizes like this is not responsive. We should be using flexbox to define a ratio